sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1871,12 +1871,15 @@ def clip(self, axis, **kwargs):
     """lower and upper must be :class:`DeferredSeries` instances, or constants.
     array-like arguments are not supported as they are order-sensitive."""
```

Comment:
"array-like arguments are not supported as they are order-sensitive." -> "Array-like arguments are not supported because they are order-sensitive."

Reply:
Done